### PR TITLE
rake task to run specs with all browsers

### PR DESCRIPTION
**Pull request by titusfortner**
```rake all_browsers``` works to run all applicable tests for all browsers applicable for the operating system. I'm not a rakefile expert, so this probably can be improved.

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -121,3 +121,44 @@ end
...
+
+def windows?
+  require 'selenium-webdriver'
+  Selenium::WebDriver::Platform.os == :windows
+end
+
+desc 'Run specs in all browsers'
+task all_browsers: [:firefox,
+                    :chrome,
+                    (:safari if mac?),
```
Comment (p0deje) [inline Rakefile]: You can avoid defining custom methods:

```ruby
(:safari if Selenium::WebDriver::Platform..mac?)
```

File: Rakefile [hunk 2]
```diff
@@ -121,3 +121,44 @@ end
...
+                    (:ie if windows?),
```
Comment (p0deje) [inline Rakefile]: You haven't defined `:ie` task

**Comment by p0deje** [comment]
What do you think about defining `:spec` as namespace so we could use it like:

```bash
$ rake spec:firefox # default
$ rake spec:chrome
# so on
$ rake spec:all_browsers
```

**Comment by titusfortner** [comment]
Thanks for the help, it is much cleaner now.

**Pull request merged**